build_ngtf.py

```diff
@@ -216,6 +218,9 @@ def main():
     if arguments.use_prebuilt_tensorflow != '':
         tf_version = arguments.use_prebuilt_tensorflow
 
+    if arguments.use_intel_tensorflow != '':
+        use_intel_tf = True
```

Comment:
No need for this variable, use `arguments.use_intel_tensorflow` directly.

Reply:
This variable is needed for  build_tensorflow and build_tensorflow_cc functions to build with config = mkl option.